src/ocean/util/serialize/contiguous/Serializer.d

```diff
@@ -54,14 +54,21 @@ struct Serializer
 
     /***************************************************************************
 
-        Serializes the data in s
+        Serializes the data in src.
+
+        Note that the serialization process may expand the dst buffer more than
+        ultimately necessary, but always returns a slice into the buffer of
+        only the required length.
```

Comment:
To be more specific, it return slice that covers bytes of struct instance itself, not including any of indirections (that are placed after). Can you please mention that?

Reply:
Yes, sure. I'll add your suggestion and also add the line `Calling applications should be careful to use the returned buffer rather than the dst buffer directly`  if it sounds good to you.